resources/views/transfers/_transfers.blade.php

```diff
@@ -18,7 +18,7 @@
     <tbody>
       <?php $index = 0; ?>
       @foreach ($transfers as $transfer)
-      <tr role="button" class="@if ($marks->contains($transfer->id)) marked @endif" data-id="{{ $transfer->id }}" data-index="{{ $start + $index }}">
+      <tr role="button" class="@if ($marks->contains($transfer->id)) marked @endif" data-id="{{ $transfer->id }}" data-index="{{ $start + $index }}" data-sort-column="{{$sortColumn}}" data-sort-direction="{{$sortDirection}}">
```

Comment:
Does this also need an `engineerLastName` check? I also wonder if it would make more sense to set these attributes on the table itself rather than repeating them for every row.

Reply:
i'm not sure what you mean by " `engineerLastName` check" could you explain that a little more? and i'll try putting the present sort and direction on the table itself. it was just easier to do this initially on every row so i didn't have to traverse the DOM

Reply:
Sorry, I was just referring to the ternary like the other comment. That makes sense. It may still be worth putting these attributes in one place, though.

Reply:
yeah, makes sense! will try now